script.js

```diff
@@ -1,12 +1,375 @@
-//You can edit ALL of the code here
+window.onload = setup
+
+const shows = getAllShows()
+let allEpisodes
+let selectedShowId
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+    loadAllShows()
+    makePageForShows()
+    searchFunc()
+}
+
+const rootElem = document.getElementById('root')
+    //a container for all episodes
+let divContainer = createHtmlElement('div')
+divContainer.className = 'container'
+    //Create and append select show list
+let showList = createHtmlElement('select')
+showList.className = 'showSelect'
+showList.addEventListener('change', searchShow)
+    //Create and append select episode list
+let episodeSelectList = createHtmlElement('select')
+episodeSelectList.className = 'episodeSelect'
+episodeSelectList.addEventListener('change', searchEpisode)
+    // create a legend for search boxes
+let searchBox = createHtmlElement('legend')
+searchBox.innerHTML = 'Search'
+searchBox.appendChild(showList)
+searchBox.appendChild(episodeSelectList)
+rootElem.appendChild(searchBox).className = 'search'
+let searchInput = createHtmlElement('input')
+searchInput.placeholder = 'your search item...'
+searchBox.appendChild(searchInput)
+searchInput.setAttribute('type', 'text')
+let searchLabel = createHtmlElement('h3')
+searchLabel.className = 'searchEpisode'
+searchBox.appendChild(searchLabel)
+    //go back navigation button
+let goBackButton = createHtmlElement('button')
+goBackButton.className = 'back-button'
+goBackButton.innerText = 'Home'
+searchBox.appendChild(goBackButton)
+rootElem.appendChild(divContainer)
+
+goBackButton.addEventListener('click', () => {
```

Comment:
here, you have a bug, if I select a show and then I press home btn the selected show it's still there in the select form.